tests/integration/it_floors_test.py

```diff
@@ -65,5 +64,8 @@ def test_edit(client, auth):
 @pytest.mark.skip(reason="auth.login() is not yet implemented")
 def test_delete(client, auth):
     auth.login()
-    response = client.post("/floors/delete", data={"id": 1})
+    floor = factories.FloorFactory()
+    response = client.post(url_for("floor.delete", id=floor.id))
     assert response.headers["Location"] == "http://localhost/floors/"
+    assert response.status_code == HTTPStatus.FOUND
+    assert Floor.query.count() == 0
```

Comment:
@emilianodellacasa Can you also add a "negative" test? What happens if the delete is called with no authenticated user? They should be redirected to login, right?

Reply:
@amihaiemil There was one, now I fixed it! Thanks for pointing it out